ui-tests/src/test/java/io/syndesis/qe/steps/customizations/extensions/ExtensionSteps.java

```diff
@@ -130,10 +127,9 @@ public void uploadExtensionFromFile(String extensionName, String extensionPath)
 
         String techExtensionUrl = extensionPath + extensionName;
         Path techExtensionJar = Paths.get(techExtensionUrl).toAbsolutePath();
-        $(By.cssSelector("input[type='file']")).shouldBe(visible).uploadFile(techExtensionJar.toFile());
+        $(By.tagName("input")).should(exist).uploadFile(techExtensionJar.toFile());
```

Comment:
don't we also want it to be visible?

Reply:
No, it is not visible in react ui, its a hidden element :/

Reply:
hang on, so how does the user upload the file?

Reply:
Drag'n'drop area != input element. We need the input element so we can call upload method on it. Don't waste your time on WIP draft PRs ;-)

Reply:
> Don't waste your time on WIP draft PRs ;-)

Well, don't request a review on them then.

Reply:
Hmm I thought that the review email is not sent unless it is not a draft. My bad then.